roles/postgresql/templates/init.sh.j2

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+{% for user in postgresql_users %}
+psql -v ON_ERROR_STOP=1 --username "postgres" <<-EOSQL
+    CREATE USER {{ user.name }} WITH PASSWORD '{{ user.password }}';
+EOSQL
+{% endfor %}
+
+{% for db in postgresql_databases %}
+psql -v ON_ERROR_STOP=1 --username "postgres" <<-EOSQL
+    CREATE DATABASE {{ db.name }};
+    ALTER DATABASE {{ db.name }} OWNER TO {{ db.owner }};
```

Comment:
Per https://www.postgresql.org/docs/current/sql-createdatabase.html you can set the owner on creation
```suggestion
    CREATE DATABASE {{ db.name }} OWNER {{ db.owner }};
```